### DISCUSSION TITLE
Cut off comment boxes in material for mkdocs' blog posts

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#6122] [General]
Noticed this a while ago. Can't tell if it is a general problem with the theming of Giscuss itself, or if it is something on Material for MkDocs' side...

But the comment section is slightly "cut off" on the right side:
![grafik](https://github.com/squidfunk/mkdocs-material/assets/11576465/6f3492d6-c4a5-4b6e-8588-304700f29dda)

**Comment by squidfunk** [comment]
This is new and related to Giscus. We don't mess with their stylesheets, and the container just stretches to the available width. Please report this upstream to the maintainers of Giscus, so they know about it and can fix it.
